elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -100,6 +127,11 @@ private IndexManagementStrategy getIndexManagementStrategy(Properties properties
 		return strategy != null ? IndexManagementStrategy.valueOf( strategy ) : IndexManagementStrategy.NONE;
 	}
 
+	private String getIndexManagementWaitTimeout(Properties properties) {
+		String timeout = properties.getProperty( ElasticsearchEnvironment.INDEX_MANAGEMENT_WAIT_TIMEOUT );
+		return timeout != null ? Integer.valueOf( timeout ) + "ms" : DEFAULT_INDEX_MANAGEMENT_WAIT_TIMEOUT;
```

Comment:
Ok, I'll use that helper.

Let's not go overboard with this stuff, though. The prop is documented, and people will get a meaningful exception when giving something else than a number. If someone configures a negative timeout, I think I don't care ;)